syntax = "proto3";


import "notification/notification_message.proto";
import "queue/queue_message.proto";

package notification;
option go_package = "/proto_files/notification"; //REPLACE WITH YOUR GITHUB PROJECT DIRECTORY PATH

service NotificationService {
  rpc CreateNotification(CreateNotificationRequest) returns (CreateNotificationResponse) {};
}

message CreateNotificationRequest {
  queue.Ticket user_ticket = 1;
  optional int64 current_queue_number = 2;
  string notification_type = 3;
}

message CreateNotificationResponse {
  Notification notification = 1;
}

